grantfinancialsupport.civix.php

```diff
@@ -6,10 +6,10 @@
  * The ExtensionUtil class provides small stubs for accessing resources of this
  * extension.
  */
-class CRM_Grantfinancialsupport_ExtensionUtil {
-  const SHORT_NAME = "grantfinancialsupport";
-  const LONG_NAME = "biz.jmaconsulting.agbu.grantfinancialsupport";
-  const CLASS_PREFIX = "CRM_Grantfinancialsupport";
+class CRM_GrantFinancialSupport_ExtensionUtil {
```

Comment:
Change of case here looks like it might cause a problem perhaps

Reply:
Thanks for pointing that. Updated the PR